packages/vaex-core/vaex/__init__.py

```diff
@@ -229,6 +229,21 @@ def from_astropy_table(table):
     return vaex.file.other.DatasetAstropyTable(table=table)
 
 
+def from_dict(data):
+    """Create an in memory dataset from a dict with column names as keys and list/numpy-arrays as values
+
+    Example
+
+    >>> data = {'A':[1,2,3],'B':['a','b','c']}
+    >>> dataset = vaex.from_dict(data)
```

Comment:
Please also show the output of the example, you can see for instance the example of `DataFrame.sort` to guide you.